visual-tests/test-scripts/series/stacked-bar.js

```diff
@@ -9,17 +9,38 @@
 
     d3.csv('stackedBarData.csv', function(error, data) {
 
-        var categories = data.slice(1).map(function(d) { return d.State; });
-        var yMax = d3.max(data, function(d) {
-            var total = 0;
-            for (var prop in d) {
-                if (prop !== 'State') {
-                    total += Number(d[prop]);
+        var categories = data.map(function(d) { return d.State; });
+
+        // Format data into a map of series key to series data & a data array to pass to our stacked-bar series.
```

Comment:
I wonder if it might be worth creating a utility from this? I.E. something to convert a typical dataset, which is an array of objects (like that created by our data generator), into the format used by the stack layout:
```js
var dataset = [
    [
        { x: 0, y: 5 },
        //...
    ],
    //...
];
```